### ISSUE TITLE
JaSerializer's ContentTypeNegotiation plug is invalid

### ISSUE THREAD

**Issue by hunterboerner** [AgilionApps/ja_serializer#57]
The current [content_type_negotiation.ex][] requires the client to specify an `Accept` header containing the JSON-API media type. however, the [JSON-API spec][] only requires that _if_ the `Accept` header is specified, it must contain a JSON-API media typ type without any media type params. I.e. the server should not be returning a 4xx error code if no `Accept` header is set.

[content_type_negotiation.ex]: https://github.com/AgilionApps/ja_serializer/blob/17cdad1058a83b59c50dd672fd57b65aadf35e28/lib/ja_serializer/content_type_negotiation.ex

[JSON-API spec]: http://jsonapi.org/format/#content-negotiation-clients

**Comment by alanpeabody** [comment]
Thanks @hunterboerner for some reason I find this particular section hard to follow. I will take a look at this with fresh eyes this week.
